lib/core.js

```diff
@@ -20,6 +21,33 @@ function SealiousCore(path_to_package_json, mode, layer_name){
 
 SealiousCore.prototype = new function(){
 
+	var check_version = function(package_info){
+		var pkg = require("../package.json")
+		var arraySplit = pkg.version.split(".");
+		var version = arraySplit[0]+"."+arraySplit[1];
+		var url = "http://registry.npmjs.org/sealious/"+version;
+
+		http.get(url, function(res) {
+			res.setEncoding('utf8');		
+			var body = "";
+			res.on('data', function (chunk) {
+				body += chunk.toString();
+			});
+			res.on('end', function(){
+				var sealious_npm = JSON.parse(body);
+				Sealious.Logger.info("Latest Sealious version: ", sealious_npm.version);
```

Comment:
Uprośćmy troszkę - jak nasza wersja jest aktualna, umieśćmy tylko jeden komunikat -  `"Sealious [version] is up-to-date"` (w aktualnej wersji kodu są dwa komunikaty)
Jak nie mamy aktualnej wersji też proponuję jednak jeden wpis w logu -` "Sealious [version] - update available. Run 'npm install sealious@"+sealious_npm.version+"' to update."` - aktualnie pojawią się 3 wpisy